sdk/communication/AzureCommunicationChat/AzureCommunicationChat.podspec.json

```diff
@@ -35,8 +35,8 @@
     "AzureCommunicationCommon": [
       "~> 1.0"
     ],
-    "TrouterClientIos": [
-      "0.0.1-beta.4"
+    "TrouterClient": [
```

Comment:
This should be `Trouter` not `TrouterClient`

Reply:
Fixed, thanks